server/routes/contacts/manage/contact-details/contactDetailsController.test.ts

```diff
@@ -268,6 +268,69 @@ describe('GET /contacts/manage/:contactId', () => {
       expect($('.relationship-active-value').text().trim()).toStrictEqual('No')
       expect($('.relationship-comments-value').text().trim()).toStrictEqual('Not provided')
     })
+
```

Comment:
I am hoping I can update this for other scenrios and reuse,

Reply:
I will update this for relationship, next of kin , comments , active scenarios, just need to pass the right json object only, 

This test does not do much - only it is testing true or false values showing YES or NO correctly, Mostly repetitive code,